backpack/core/derivatives/crossentropyloss.py

```diff
@@ -1,9 +1,13 @@
 """Partial derivatives for cross-entropy loss."""
+from itertools import product as itertools_product
 from math import sqrt
-from typing import List, Tuple
+from typing import Callable, Dict, Iterable, List, Tuple, Union
 
-from torch import Tensor, diag, diag_embed, einsum, multinomial, ones_like, softmax
+from einops import rearrange
+from numpy import prod
```

Comment:
```suggestion
from math import prod
```
This would be nicer but doesn't work. (Import error)